end-to-end-tests/src/test/java/tech/pegasys/peeps/consensus/qbft/quorumbesu/QbftWithGasAndBlockRewardTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.peeps.consensus.qbft.quorumbesu;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import tech.pegasys.peeps.FixedSignerConfigs;
+import tech.pegasys.peeps.NetworkTest;
+import tech.pegasys.peeps.network.ConsensusMechanism;
+import tech.pegasys.peeps.network.Network;
+import tech.pegasys.peeps.node.Account;
+import tech.pegasys.peeps.node.Web3Provider;
+import tech.pegasys.peeps.node.Web3ProviderType;
+import tech.pegasys.peeps.node.verification.ValueReceived;
+import tech.pegasys.peeps.node.verification.ValueSent;
+import tech.pegasys.peeps.signer.SignerConfiguration;
+
+import java.util.List;
+
+import org.apache.tuweni.crypto.SECP256K1.KeyPair;
+import org.apache.tuweni.eth.Address;
+import org.apache.tuweni.units.ethereum.Wei;
+import org.junit.jupiter.api.Test;
+
+public class QbftWithGasAndBlockRewardTest extends NetworkTest {
+  private final Wei gasPrice = Wei.valueOf(1);
```

Comment:
What would happen if this was 0? Would you expect positive price gas txs to work in the same way? Obviously the tx rewards would be different.

Reply:
I imagine this would break besu if we ever upgraded the PEEPS genesis file to londonBlock because gasPrice < baseFee. Probably not an issue to worry about in this PR though